build.gradle.kts

```diff
@@ -1,7 +1,9 @@
 plugins {
     id("org.openrewrite.build.recipe-library") version "latest.release"
     id("org.openrewrite.build.moderne-source-available-license") version "latest.release"
+    id("com.netflix.nebula.provided-base") version "10.0.1"
 }
+apply(plugin = "com.netflix.nebula.provided-base")
```

Comment:
You shouldn't have both a plugin `id` block above and _also_ an `apply` call. This `apply` mechanism is an old way of applying plugins.

Reply:
Thx for the info! I am still trying to make it work; I don't have much experience with Gradle, so this was my first attempt. At the moment, the `provided` functions do not work at all 😞.

Reply:
Ah, now I get it! The plugin does not do anything anymore, it's only an [empty](https://github.com/nebula-plugins/gradle-extra-configurations-plugin/blob/main/src/main/groovy/nebula/plugin/extraconfigurations/ProvidedBasePlugin.groovy) plugin today:
> LOGGER.warn("nebula.provided-base plugin does nothing. Use the 'compileOnly' configuration")

See alse: [history](https://github.com/nebula-plugins/gradle-extra-configurations-plugin/commits/main/src/main/groovy/nebula/plugin/extraconfigurations/ProvidedBasePlugin.groovy).

What do you propose now @jkschneider? Start going for the `compileOnly` scope? What do you think @timtebeek ?

---
FYI: there is another [archived project](https://github.com/spring-attic/propdeps-plugin) with the same intention.